modules/ROOT/pages/prereq-env.adoc

```diff
@@ -0,0 +1,42 @@
+= Environment Limitations
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Adhere to the following recommendations and limitations when setting up your environment and deploying apps:
+
+* Environment:
+
+** Production: 6 nodes of 8 cores and 32GB 
+** Maximum number of servers: 500
```

Comment:
All these belong below under `Runtime Management`

Reply:
Moved - thank you!